client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx

```diff
@@ -4,28 +4,59 @@ import GridList from '@material-ui/core/GridList';
 import GridListTile from '@material-ui/core/GridListTile';
 import GridListTileBar from '@material-ui/core/GridListTileBar';
 import IconButton from '@material-ui/core/IconButton';
+import { useAuth } from '../../context/useAuthContext';
 import EmojiEventsIcon from '@material-ui/icons/EmojiEvents';
+import CircularProgress from '@material-ui/core/CircularProgress';
 import useStyles from './useStyles';
 // mock data 4 now
-import { Contest, Contests } from '../../interface/tempContestData';
+//import { Contest as Contesto, Contests } from '../../interface/tempContestData';
 import { Submissions } from '../../interface/tempSubmissionData';
+import { getContestById } from '../../helpers/APICalls/contest';
+import { RouteComponentProps, useParams } from 'react-router-dom';
+import { Contest } from '../../interface/Contest';
 // Used to test the two views. 1 is the id of the contest owner and shows the owner view(view all submissions, pick winner). 2 shows the submittor view(contest deets and submit button)
 const contestOwnerId = 11;
 
+interface RouteParams {
+  id: string;
+}
+
 export default function ViewContestSubmissions(): JSX.Element {
   const classes = useStyles();
   const [contestCard, setContestCard] = useState<Contest>(Object);
   const [winnerIndex, setWinnerIndex] = useState(-1);
+  const { loggedInUser } = useAuth();
+
+  const params = useParams<RouteParams>();
 
+  console.log(params);
   //   pulling dummy data for now
-  const handleContest = () => {
-    setContestCard(Contests[0]);
-  };
+  // const handleContest = () => {
+  //   setContestCard(Contests[0]);
+  // };
+
+  // useEffect(() => {
+  //   handleContest();
+  // }, [contestCard]);
```

Comment:
We don't need to push commented code into production. If it needs to be added in the future you could just refer to this PR and bring it back.